pkg/setting/setting.go

```diff
@@ -507,10 +507,11 @@ type Cfg struct {
 	FeatureManagement FeatureMgmtSettings
 
 	// Alerting
-	AlertingEvaluationTimeout   time.Duration
-	AlertingNotificationTimeout time.Duration
-	AlertingMaxAttempts         int
-	AlertingMinInterval         int64
+	AlertingEvaluationTimeout      time.Duration
+	AlertingNotificationTimeout    time.Duration
+	AlertingMaxAttempts            int
+	AlertingMinInterval            int64
+	AlertingRuleVersionRecordLimit int
```

Comment:
These are legacy settings. We can drop it from here

Reply:
@yuri-tceretian This was my best guess. Can you point to the best place to add these settings? Thanks

Reply:
@yuri-tceretian thanks updated